code/modules/research/nodes/illegal.dm

```diff
@@ -45,20 +45,22 @@
 
 	unlocks_designs = list(/datum/design/research/item/chameleon_kit)
 
-/datum/technology/freedom_implant
-	name = "Glass Case- 'Freedom'"
-	desc = "Freedom' implant releases EM impulse in the owner body, that destroy all implants of slaving in, breaking chains."
+/datum/technology/illegal_implants
```

Comment:
A nitpick, but these implants aren't illegal, I'd find a different name for them.

Reply:
Sure, i can rename them, but tell me what name will be suitable most?